applications-frontend/src/sass/styles.scss

```diff
@@ -14,4 +49,5 @@ $navbar-item-img-max-height: 3rem;
 // 6. Import the rest of Bulma
 @import "~bulma";
 
-@import "flex";
\ No newline at end of file
+@import "flex";
```

Comment:
I would import these first.

CSS is cascading so anything you define above this import could be overwritten by `bulma` if they share a common selector.

You probably want to import bulma and then add custom styles.